LICENSE

```diff
@@ -295,9 +295,21 @@ Files:
 
     Flot bundles additional works:
 
-    jquery.pie.js
-    * Created by Brian Medendorp, June 2009
-    * Updated November 2009 with contributions from: btburnett3, Anthony Aragues and Xavi Ivars
+    jquery.flot.pie.js
+        Flot plugin for rendering pie charts.
+
+        Copyright (c) 2007-2014 IOLA and Ole Laursen.
+        Licensed under the MIT license.
+
+        The plugin assumes that each series has a single data value, and that each
+        value is a positive integer or zero.  Negative numbers don't make sense for a
+        pie chart, and have unpredictable results.  The values do NOT need to be
+        passed in as percentages; the plugin will calculate the total and per-slice
+        percentages internally.
+
```

Comment:
This is not part of the license/attribution content. It's a description of the plugin, which does not need to be included here. It is unfortunate they intermingled the description with the attribution notices. I would snip out the description from the middle of this in our files.